src/Sylius/Bundle/WebBundle/Resources/views/Frontend/layout.html.twig

```diff
@@ -134,7 +134,7 @@
         {% block footer %}
             <div class="footer">
                 <p class="text-muted">
-                    &copy; Sylius, 2011 - {{ 'now'|date('Y') }}.
+                    &copy; Sylius, 2011 - {{ 'now'|sylius_localized_date('Y') }}.
```

Comment:
Yeah, let's use the text, not a date and copyright. :) We will tweak it anyway with reworked admin theme.